lib/Charts/ChartData.js

```diff
@@ -53,7 +53,7 @@ var ChartData = function(points, parameters) {
      * Units of this set of points.
      * @type {String}
      */
-    this.units = parameters.units;
+    this.units = parameters.units || 'unknown';
```

Comment:
Unfortunately adding "unknown" here then displays "unknown" in the y-axis label, eg.
![image](https://cloud.githubusercontent.com/assets/1757858/22867891/b56ea21c-f1e3-11e6-84dc-db9a3aa90cff.png)
![image](https://cloud.githubusercontent.com/assets/1757858/22867895/c0bd673e-f1e3-11e6-86c0-50973a3ba7ba.png)
(these from http://localhost:3001/#build/terriajs/test/init/charts.json)